modules/efk-logging-elasticsearch-exposing.adoc

```diff
@@ -96,14 +88,18 @@ route.route.openshift.io/elasticsearch created
 .. Get the token of this ServiceAccount to be used in the request:
 +
 ----
-$ token=$(oc whoami -t)
```

Comment:
I think we want to keep this setting token since it is used in the curl command line at the line 97.